src/PresentationalComponents/RulesTable/RulesTable.js

```diff
@@ -73,7 +73,7 @@ const RulesTable = ({ rules, filters, rulesFetchStatus, setFilters, fetchRules,
 
     const fetchRulesFn = useCallback(() => {
         urlBuilder(filters, selectedTags, workloads);
-        let options = selectedTags.length && ({ tags: selectedTags.map(tag => encodeURIComponent(tag)).join('&tags=') });
```

Comment:
@AllenBW not sure if this an issue or not 🤔 but it looks like the tags are getting joined with `,` instead of with `&tags=` in this implementation

Reply:
yeah they are, not an issue, api supports it both ways, cyndi enables support for &tags=

Reply:
well that's awesome 😁 💯